docs/preview/features/databricks/databricks-info-provider.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Interact with Databricks to gain insights"
+layout: default
+---
+
+# Interact with Databricks to gain insights
+
+## Installation
+
+To use these features, you have to install the following package:
+
+	```shell
```

Comment:
Can we remove the tabs here? I accidentally added these for the GH suggestion

Reply:
Yes.